### PR TITLE
DEV: Rearrange command line interface to simplify addition of new commands

### PR DESCRIPTION
**Pull request by shaneahmed**
- Move all cli commands/options from cli.py to cli_commands to make it clean and easier to add new commands
- Fix test in tests/test_slide_info
- Move string_to_tuple to tiatoolbox/utils/misc.py
- Fix docstring in patch_predictor.

### CODE REVIEW

**Review by John-P — COMMENTED** [review]
Looks good. Just a few comments. Just wanted to ask why you are using `tiatoolbox/cli_commands/...`? It seems a bit uneccesarrily verbose. Why not use `tiatoolbox/cli/...` and put the `cli.py` stuff the `__init__.py`?

File: tiatoolbox/cli_commands/tissue_mask.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+
+@main.command()
+@click.option("--img_input", help="Path to WSI file")
+@click.option(
+    "--output_path",
+    help="Path to output file to save the image region in save mode,"
+    " default=tissue_mask",
+    default="tissue_mask",
+)
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply tiatoolbox/cli_commands/tissue_mask.py:26]: I would prefer with `--` as the list can be long and keeping track of positional arguments can be difficult sometimes. It also helps reduce errors while running through cli.

**Comment by shaneahmed** [comment]
> Looks good. Just a few comments. Just wanted to ask why you are using `tiatoolbox/cli_commands/...`? It seems a bit uneccesarrily verbose. Why not use `tiatoolbox/cli/...` and put the `cli.py` stuff the `__init__.py`?

I will change this. I agree with your suggestion.

File: tiatoolbox/utils/misc.py [hunk 2]
```diff
@@ -671,3 +671,8 @@ def model_to(on_gpu, model):
         model = model.to("cpu")
 
     return model
+
+
+def string_to_tuple(file_types):
+    """Split file_types string to tuple."""
+    return tuple(file_types.split(", "))
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply tiatoolbox/utils/misc.py:678]: Can we do it in a way that it supports both with and without space?

[the remark this answers is not in the document]
Comment (mostafajahanifar) [reply tiatoolbox/utils/misc.py:678]: You can do it using `re` package:
```
import re
def string_to_tuple(file_types):
    """Split file_types string to tuple."""
    return tuple(re.split(', |,', file_types))
```

File: tiatoolbox/utils/misc.py [hunk 3]
```diff
@@ -23,6 +23,7 @@
 import json
 import os
 import pathlib
+import re
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply tiatoolbox/utils/misc.py]: Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/156/commits/fb3eb25abe3abef9fc117093235cc8d6abeae833

File: tiatoolbox/utils/misc.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (shaneahmed) [reply tiatoolbox/utils/misc.py]: Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/156/commits/790251658d217da9df5681fae4e0d7d206c1655e

**Pull request merged**